project/src/attachments/Camera.gd

```diff
@@ -37,6 +37,15 @@ var fps = 0
 var vflip: bool = false
 var hflip: bool = false
 
+signal cam_locked
+signal cam_freed
+
+var locked_cam: Spatial = null
```

Comment:
might not to be lock_cam ?

Reply:
locked_cam is the original variable name used in the CamControls, so I just kept it as is.